pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -0,0 +1,87 @@
+package mutators
+
+import (
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/konveyor/forklift-controller/pkg/forklift-api/webhooks/util"
+	"github.com/konveyor/forklift-controller/pkg/lib/logging"
+	admissionv1 "k8s.io/api/admission/v1beta1"
+	core "k8s.io/api/core/v1"
+)
+
+var log = logging.WithName("mutator")
+
+type OvirtCertMutator struct {
+}
+
+func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admissionv1.AdmissionResponse {
+	log.Info("secret mutator was called")
+	raw := ar.Request.Object.Raw
+	secret := &core.Secret{}
+	err := json.Unmarshal(raw, secret)
+	if err != nil {
+		log.Error(err, "mutating webhook error")
+		util.ToAdmissionResponseError(err)
+	}
+
+	if _, ok := secret.GetLabels()["createdForProviderType"]; ok && secret.GetLabels()["createdForProviderType"] == "ovirt" {
```

Comment:
why querying "createdForProviderType" twice? would the following work
```suggestion
	if type, ok := secret.GetLabels()["createdForProviderType"]; ok && type == "ovirt" {
```